data_override/include/data_override.inc

```diff
@@ -1107,48 +1362,228 @@ subroutine DATA_OVERRIDE_3D_(gridname,fieldname_code,return_data,time,override,d
+
       else
         !> If this in an ongrid case and you are not in the compute domain, send in `data` to be the correct
         !! size
-        call time_interp_external(id_time,time,return_data(startingi:endingi,startingj:endingj,1),verbose=.false., &
-                               is_in=is_in,ie_in=ie_in,js_in=js_in,je_in=je_in,window_id=window_id)
+
+        if (multifile) then   ! bridging between files
+          if (time<first_record) then
+            ! previous file must be init and time must be between last record of previous file and
+            ! first record of current file
+            if (id_time_prev<0) call mpp_error(FATAL,'data_override:previous file needed with multifile')
+            prev_dims = get_external_field_size(id_time_prev)
+            if (time<data_table(index1)%time_prev_records(prev_dims(4))) call mpp_error(FATAL, &
+                'data_override: time_interp_external_bridge should only be called to bridge with previous file')
+            ! bridge with previous file
+            call time_interp_external_bridge(id_time_prev,id_time,time,&
+                                             return_data(startingi:endingi,startingj:endingj,1),verbose=.false., &
+                                             is_in=is_in,ie_in=ie_in,js_in=js_in,je_in=je_in,window_id=window_id)
+          elseif (time>last_record) then
+            ! next file must be init and time must be between last record of current file and
+            ! first record of next file
+            if (id_time_next<0) call mpp_error(FATAL,'data_override:next file needed with multifile')
+            if (time>data_table(index1)%time_next_records(1)) call mpp_error(FATAL, &
+                'data_override: time_interp_external_bridge should only be called to bridge with next file')
+            ! bridge with next file
+            call time_interp_external_bridge(id_time,id_time_next,time,&
+                                             return_data(startingi:endingi,startingj:endingj,1),verbose=.false., &
+                                             is_in=is_in,ie_in=ie_in,js_in=js_in,je_in=je_in,window_id=window_id)
+          else  ! first_record <= time <= last_record, do not use bridge
+            call time_interp_external(id_time,time,return_data(startingi:endingi,startingj:endingj,1),verbose=.false., &
+                                      is_in=is_in,ie_in=ie_in,js_in=js_in,je_in=je_in,window_id=window_id)
+          endif
+        else  ! standard behavior
+          if ((time<first_record) .or. (time>last_record)) then
+            call mpp_error(WARNING, &
+                 'data_override: current time outside bounds, use [previous]:current:[next] files in data_table')
+          endif
+          call time_interp_external(id_time,time,return_data(startingi:endingi,startingj:endingj,1),verbose=.false., &
+                                    is_in=is_in,ie_in=ie_in,js_in=js_in,je_in=je_in,window_id=window_id)
+        endif  ! end bridge
```

Comment:
This end bridge comment is confusing because there are two if blocks documented at bridging